docs/api.md

```diff
@@ -475,6 +475,52 @@ Renders a link that points to the LoginRoute or `/login` if no LoginRoute is spe
 <LoginLink><img src="wrap-something-in-a-login-link.png" /></LoginLink>
 ```
 
+#### SocialLoginLink
+
+Renders a link that can be used to sign in with a social provider.
+
+```html
+<SocialLoginLink providerId='facebook' />
+<SocialLoginLink providerId='facebook'>Sign in with Facebook</SocialLoginLink>
+```
+
+Renders a link that can be used to sign into a social provider.
+
+```html
+<SocialLoginLink providerId='facebook' />
+```
+
+Set specific scopes by providing the `scope` option.
+
+```html
+<SocialLoginLink providerId='facebook' scope='email user_friends' />
+```
+
+Set your own redirect URI by providing the `redirectUri` option. If this isn't set then it defaults to `[protocol]://[host]/callbacks/[providerId]`.
+
+```html
+<SocialLoginLink providerId='facebook' redirectUri='http://www.my-domain.com/callbacks/facebook' />
+```
+
+#### SocialLoginButton
+
+Renders a button that can be used to sign in with a social provider.
+
+```html
+<SocialLoginButton providerId='facebook' />
+<SocialLoginButton providerId='facebook'>Sign in with Facebook</SocialLoginButton>
+```
+
+If you don't want the button to show an icon, then simply disable it by setting the `hideIcon` option.
+
+```html
+<SocialLoginButton providerId='facebook' hideIcon={ true } />
+```
+
+**Note:** The same options that are supported by the `SocialLoginLink` component are also supported by this.
+
+**Important:** This component relies on [Font Awesome](http://fortawesome.github.io/Font-Awesome/) in order to render icons for the various social providers. So if you want to use this button with icons, you also need to install Font Awesome on your site.
```

Comment:
It's actually added, didn't you try it with the `social-login` branch? :)
https://github.com/stormpath/stormpath-express-react-example/tree/social-login/build/fonts